### PR TITLE
Updated User Guide.

### PR DESCRIPTION
**Pull request by pv101**
First shot at implementing notes Niki gave me last week.

### CODE REVIEW

File: doc/src/user_guide.rst [hunk 1]
```diff
@@ -494,13 +498,13 @@ Parameterises the time-integration scheme used by the solver with
 
                *int*
 
-            - ``pseudo-aresid`` --- absolute residual tolerance
+            - ``pseudo-residtol`` --- pseudo residual tolerance
```
Comment (pv101) [inline doc/src/user_guide.rst]: Should this be pseudo-resid-tol ?

File: doc/src/user_guide.rst [hunk 2]
```diff
@@ -494,13 +498,13 @@ Parameterises the time-integration scheme used by the solver with
...
 
                *float*
 
-            - ``pseudo-rresid`` --- relative residual tolerance
+            - ``resid-norm`` --- pseudo residual norm
```
Comment (pv101) [inline doc/src/user_guide.rst]: Should this be pseudo-resid-norm ?

**Comment by pv101** [comment]
When using ac-euler or ac-navier-stokes and the sampler plugin, do format=primative and format=conservative both work, and return the same thing. Or does one fail?

File: doc/src/user_guide.rst [hunk 3]
```diff
@@ -359,7 +359,19 @@ Parameterises the solver with
...
 1. ``system`` --- governing system:
 
-    ``euler`` | ``navier-stokes``
+    ``euler`` | ``navier-stokes`` | ``ac-euler`` | ``ac-navier-stokes``
+
+    where
+
+    ``euler`` | ``navier-stokes`` require
+
+        - ``viscosity-correction`` --- viscosity correction:
```
Comment (FreddieWitherden) [inline doc/src/user_guide.rst]: No.

**Comment by nloppi** [comment]
Both primitive and conservative format work and return the same thing.

File: doc/src/user_guide.rst (hunk 2 above)
Comment (nloppi) [inline doc/src/user_guide.rst]: Yes, pseudo-resid-norm makes more sense. It has been changed in my pull request.

File: doc/src/user_guide.rst (hunk 1 above)
Comment (nloppi) [inline doc/src/user_guide.rst]: Changed to pseudo-resid-tol in the code.

**Comment by nloppi** [comment]
I also forgot to mention that in ac-navier-stokes,  kinematic viscosity nu needs to be specified.

[constants]
mu — dynamic viscosity for navier-stokes:
 float

nu — kinematic viscosity for ac-navier-stokes:
 float

File: doc/src/user_guide.rst [hunk 4]
```diff
@@ -1124,7 +1181,8 @@ with
 
            *float* | *string*
 
-    ``no-slp-isot-wall`` requires
+    ``no-slp-isot-wall`` only works with ``navier-stokes`` |
+    ``ac-navier-stokes`` and requires
```
Comment (nloppi) [inline doc/src/user_guide.rst]: The artificial compressibility wall BCs are called just no-slip-wall (option to specify u,v,w) and slip-wall.

File: doc/src/user_guide.rst [hunk 5]
```diff
@@ -321,27 +321,38 @@ Example::
...
     *float*
 
-2. ``mu`` --- dynamic viscosity:
+3. ``nu`` --- kinematic viscosity for ``ac-euler`` |
+   ``ac-navier-stokes``:
 
     *float*
 
-3. ``Pr`` --- Prandtl number:
+4. ``Pr`` --- Prandtl number for ``euler`` | ``navier-stokes``:
```
Comment (FreddieWitherden) [inline doc/src/user_guide.rst]: Not for Euler!

File: doc/src/user_guide.rst [hunk 6]
```diff
@@ -321,27 +321,38 @@ Example::
...
-Sets constants used in the simulation with
+Sets constants used in the simulation
 
-1. ``gamma`` --- ratio of specific heats:
+1. ``gamma`` --- ratio of specific heats for ``euler`` |
+   ``navier-stokes``:
+
+    *float*
+
+2. ``mu`` --- dynamic viscosity for ``euler`` | ``navier-stokes``:
```
Comment (FreddieWitherden) [inline doc/src/user_guide.rst]: Not for Euler!

File: doc/src/user_guide.rst [hunk 7]
```diff
@@ -321,27 +321,38 @@ Example::
...
 
     *float*
 
-2. ``mu`` --- dynamic viscosity:
+3. ``nu`` --- kinematic viscosity for ``ac-euler`` |
```
Comment (FreddieWitherden) [inline doc/src/user_guide.rst]: Not for Euler!

**Pull request closed without merging**